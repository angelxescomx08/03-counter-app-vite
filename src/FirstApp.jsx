import React from 'react'

import PropTypes from 'prop-types'

const newMessage = 'Ángel';

export const FirstApp = ({title,subtitle}) => {
    return (
        <>
            <h1>{title}</h1>
            <p>{subtitle}</p>
        </>
    )
}

FirstApp.propTypes = {
    title: PropTypes.string.isRequired,
    subtitle: PropTypes.string.isRequired
}

FirstApp.defaultProps = {
    title: 'Hola soy Goku',
    subtitle: 'No hay subtitulo'
}