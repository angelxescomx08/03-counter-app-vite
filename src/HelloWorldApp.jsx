export function HelloWorldApp(){
    return(
        <h1>Hola mundo</h1>
    )
}