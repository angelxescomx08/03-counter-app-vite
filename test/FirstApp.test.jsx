import { render } from '@testing-library/react'
import { FirstApp } from '../src/FirstApp'

describe('Pruebas en <FirstApp />', () => {
    test('Debe hacer un mattch con el snapshot', () => {
        const title = 'Hola angel'
        render(<FirstApp title={title} />)
    })
})