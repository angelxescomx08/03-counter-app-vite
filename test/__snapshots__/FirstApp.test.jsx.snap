// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <FirstApp /> Debe hacer un mattch con el snapshot 1`] = `
<div>
  <h1>
    Hola angel
  </h1>
  <p>
    No hay subtitulo
  </p>
</div>
`;
